### PR TITLE
Arc multi

### PR DESCRIPTION
**Pull request by hectorhuertas**

### CODE REVIEW

**Review by asiyani — APPROVED** [review]

**Review by asiyani — DISMISSED** [review]

File: arc/runner/common-values.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        storage: 5Gi
+
+template:
+  spec:
+    # https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/troubleshooting-actions-runner-controller-errors#error-access-to-the-path-homerunner_work_tool-is-denied
+    securityContext:
+      fsGroup: 123
+    containers:
+      - name: runner
+        image: ghcr.io/actions/actions-runner:latest
```
Comment (fillyD) [inline arc/runner/common-values.yaml:23]: should we be latest?

File: arc/runner/common-values.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    accessModes: ["ReadWriteOnce"]
+    storageClassName: "netapp-ontap-san-ext4"
+    resources:
+      requests:
+        storage: 5Gi
+
+template:
+  spec:
+    # https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/troubleshooting-actions-runner-controller-errors#error-access-to-the-path-homerunner_work_tool-is-denied
+    securityContext:
```
Comment (fillyD) [inline arc/runner/common-values.yaml:19]: some workflows use sudo, would this allow for that?

File: arc/runner/common-values.yaml (hunk 1 above)
Comment (hectorhuertas) [reply arc/runner/common-values.yaml:23]: I think so, the runner program auto-updates itself anyway, so we might as well use the latest image directly. Given that the default behaviour it's to auto-update, I don't expect breaking changes

File: arc/runner/common-values.yaml (hunk 2 above)
Comment (hectorhuertas) [reply arc/runner/common-values.yaml:19]: yes, sudo can be used inside the image, but the container is not privileged, so cannot be used to escape it

**Review by george-angel — APPROVED** [review]

**Review by fillyD — APPROVED** [review]

**Review by george-angel — APPROVED** [review]

**Review by asiyani — APPROVED** [review]

**Pull request merged**